### PR TITLE
Added classes from lesson 4

### PR DESCRIPTION
**Pull request by dimloko**
Homework for lesson_4

### CODE REVIEW

File: project/src/Autor.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+public class Autor {
+    String name; //Имя
+    String surname; //Фамилия
+    int dob; //Дата рождения
+
+    //Конструктор по умолчанию
+    public Autor() {
+        name = "Unknown";
```
Comment (RomanKashenok) [inline project/src/Autor.java]: don't assign anything if it's not required

File: project/src/Autor.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        surname = "Unknown";
+        dob = 0;
```
Comment (RomanKashenok) [inline project/src/Autor.java]: it'll be 0 by default. remove assignment

File: project/src/Book.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
+public class Book {
+    String title;
+    String autor; //Автор
+    int num_of_page; //Количество страниц
+    int year; //Год издания
+
+    //Конструктор по умлочанию
+    public Book() {
```
Comment (RomanKashenok) [inline project/src/Book.java:8]: the same comments as in Author

File: project/src/Genre.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
+public class Genre {
+    String genre_name; //Название жанра
+    int number_of_books; //Количество книг данного жанра
+    int row; //Ряд в библиотеке
+
+    //Конструктор по умолчанию
+
+    public Genre() {
```
Comment (RomanKashenok) [inline project/src/Genre.java:8]: the same as in Author

File: project/src/Library.java [hunk 5]
```diff
@@ -0,0 +1,30 @@
+public class Library {
+    String name; //Название библиотеки
+    int num_of_books; //Количество книг
+    int num_of_genres; //Количество жанров
+    int num_of_rows; //Количество рядов
+
+    //Конструктор по умолчанию
+    public Library() {
```
Comment (RomanKashenok) [inline project/src/Library.java:8]: the same as in Author

File: project/src/Library.java [hunk 6]
```diff
@@ -0,0 +1,30 @@
+public class Library {
+    String name; //Название библиотеки
+    int num_of_books; //Количество книг
```
Comment (RomanKashenok) [inline project/src/Library.java]: numOfBooks instead of num_of_books

**Review by RomanKashenok — CHANGES_REQUESTED** [review]

File: project/src/Library.java (hunk 6 above)
Comment (dimloko) [reply project/src/Library.java]: Done, hopefully right)

**Review by TheFallen058 — APPROVED** [review]

File: project/src/Genre.java [hunk 7]
```diff
@@ -1,26 +1,22 @@
 public class Genre {
     String genre_name; //Название жанра
```
Comment (RomanKashenok) [inline project/src/Genre.java]: genreName. not genre_name
and just "name" will be better

**Review by RomanKashenok — APPROVED** [review]
Fix a little according to comments and feel free t merge

**Pull request merged**